transform-service/latest/config/index.md

```diff
@@ -7,22 +7,31 @@ Use this information to configure Transform Service.
 
 ## Transform Service vs. Local and Legacy Transforms
 
-Here is a summary of the changes in the transforms introduced in Alfresco Content Services 6.2.
+Here is a summary of the changes in the transforms introduced in Alfresco Content Services 7.
 
-The Transform Service performs transformations for Content Services in Docker containers to provide greater scalability. Requests to the Transform Service are placed in a queue and processed asynchronously. Security is also improved by better isolation.
+The Transform Service and Local transformers where introduced in Alfresco Content Services 6 to help offload the 
+transformation of content to a separate process. The Legacy transforms were deprecated. In Alfresco Content Services 7, 
+the out of the box Legacy transformers and transformation framework have been removed. This helps provide greater clarity 
+around installation and administration of transformations and technically a more scalable, reliable and secure environment.
 
-* **Legacy Transforms** existed in Content Services prior to version 6.0, and ran in the same JVM as the repository. Content Services 6.2 still uses them if a rendition can't be created by the Transform Service or Local Transforms. The process of migrating custom legacy transformers is described in the Content Services documentation.
-* **Local Transforms** run in separate processes to the repository known as Transform Engines (or T-Engines for short).
+The Transform Service performs transformations for Content Services in Docker containers to provide greater scalability. 
+Requests to the Transform Service are placed in a queue and processed asynchronously. Security is also improved by better isolation.
 
-You can turn on (enable) and off (disable) Local, Legacy and Transform Service transforms independently of each other by setting Alfresco global properties. In the Content Services deployment, the Transform Service is disabled for the zip distribution but enabled for Docker Compose and Helm deployments by default. The repository will try to transform content using the Transform Service if possible, falling back to a Local Transform, and failing that, fall back to a Legacy Transform. This makes it possible to gradually migrate away from Legacy Transforms, and to take advantage of the Transform Service if it's available.
+**Local Transforms** run in separate processes to the repository known as Transform Engines (or T-Engines for short).
+
+You can turn on (enable) and off (disable) Local and Transform Service transforms independently of each other 
+by setting Alfresco global properties. In the Content Services deployment, the Transform Service is disabled for the 
+zip distribution but enabled for Docker Compose and Helm deployments by default. The repository will try to transform 
+content using the Transform Service if possible, falling back to a Local Transform. 
```

Comment:
The repository will try to transform content using the Transform Service if possible and if not it will fall back to a Local Transform.

Suggestion :-)

Reply:
Fixed,  thanks